### PR TITLE
[Fix bug 1289119] Create celery task to check for spam

### PR DESCRIPTION
**Pull request by johngian**

### CODE REVIEW

**Comment by johngian** [comment]
@akatsoulas r?

File: mozillians/users/tasks.py [hunk 1]
```diff
@@ -282,3 +283,13 @@ def remove_incomplete_accounts(days=INCOMPLETE_ACC_MAX_DAYS):
 def check_celery():
     """Dummy celery task to check that everything runs smoothly."""
     pass
+
+
+@task
```
Comment (akatsoulas) [inline mozillians/users/tasks.py]: Is this going to be a periodic task? If yes, I would suggest to schedule it here. When pr #1483 is merged we won't have the ability to do so from the admin panel.

File: mozillians/common/utils.py [hunk 2]
```diff
@@ -26,6 +23,13 @@ def absolutify(url):
 def akismet_spam_check(user_ip, user_agent, **optional):
     """Checks for spam content against Akismet API."""
 
+    AKISMET_API_KEY = getattr(settings, 'AKISMET_API_KEY', '')
+
+    if not AKISMET_API_KEY:
+        return None
+
+    AKISMET_URL = 'https://{0}.rest.akismet.com/1.1/comment-check'.format(AKISMET_API_KEY)
```
Comment (akatsoulas) [inline mozillians/common/utils.py]: Why did you move this here?

File: mozillians/users/tasks.py (hunk 1 above)
Comment (johngian) [inline mozillians/users/tasks.py]: It's not a periodic task. It gets triggered on each profile save.

File: mozillians/phonebook/views.py [hunk 3]
```diff
@@ -238,6 +239,19 @@ def get_request_data(form):
             for f in curr_forms:
                 f.save()
 
+            # Spawn task to check for spam
```
Comment (akatsoulas) [inline mozillians/phonebook/views.py]: Should we spawn this task only when the bio field is populated with data to avoid a few calculations?

Comment (johngian) [inline mozillians/phonebook/views.py]: We are checking against many fields and not just bio. Akismet might be able to check for spam even by just providing ``REMOTE_ADDR``.

**Comment by akatsoulas** [comment]
r+wc
